### DISCUSSION TITLE
Download rendered PDF client side

### DISCUSSION THREAD

**Discussion by arthurauffray** [wojtekmaj/react-pdf#1865] [Q&A]
Hi everyone, I love this project but I'm wondering if there is a way to download the rendered PDF from the client side? Some method of parsing the resulting HTML?

Thanks in advance for your input, I appreciate it!

**Comment by wojtekmaj** [comment]
Are you confusing react-pdf with @react-pdf/renderer? If so, refer to their docs.

**Reply by arthurauffray** [reply]
Hey Wojciech, I initially landed on the renderer repo but this is the correct one.
